test/FunctionalTestUtils20/FunctionalTestUtils20.csproj

```diff
@@ -3,7 +3,8 @@
   <PropertyGroup>
```

Comment:
I wonder how different test utils for `netstandard` 1 and 2? Maybe can be merged as a single project. Minor comment though